examples/client_simple.rs

```diff
@@ -27,7 +27,27 @@ fn main() {
 	let audio_file_path = args().nth(2)
 		.expect("Please specify an audio file to run STT on");
 	let dir_path = Path::new(&model_dir_str);
-	let mut m = Model::load_from_files(&dir_path.join("output_graph.pb")).unwrap();
+	let mut graph_name: String = String::from("output_graph.pb");
+	let mut scorer_name: Option<String> = None;
+	// search for model in model directory
+	for file in dir_path.read_dir().expect("Specified model dir is not a dir") {
+		if let Ok(f) = file {
+			if f.file_type().unwrap().is_file() {
+				let file_name = f.file_name().into_string().unwrap();
```

Comment:
This path to string conversion is not needed. Path has an ends_with function as well.

Reply:
I completely missed that I could get a PathBuf (woops!). I've changed it to use Path's extension function since that's exactly what I need. (ends_with seems to only work with full Path components, not parts of a filename, so I didn't end up using that)